android/java/org/chromium/chrome/browser/crypto_wallet/fragments/SwapBottomSheetDialogFragment.java

```diff
@@ -71,17 +86,27 @@ public void setupDialog(@NonNull Dialog dialog, int style) {
         mSendLayout.setOnClickListener(this);
         mSwapLayout = view.findViewById(R.id.swap_layout);
         mSwapLayout.setOnClickListener(this);
-        if (!mIsCustomNetwork) {
-            mBuyLayout.setVisibility(View.VISIBLE);
```

Comment:
have you checked does it show buy on Polygon on Binance network after such changes? As it shouldn't.

Reply:
No issue here
https://user-images.githubusercontent.com/11755381/181804956-b4bea2f9-cd2e-4429-81f1-f18ce24965b1.mp4

Reply:
I meant not Buy option, open a Buy screen and compare what we show there and what is shown in your PR

Reply:
Fixed https://github.com/brave/brave-core/pull/14397/commits/c5dc6927b21d3bce27d2c348ad2d9cc88925d0ac